### ISSUE TITLE
Chinese characters not displayed correctly

### ISSUE THREAD

**Issue by peter-jerry-ye** [flatpak/flatpak#2269]
## Linux distribution and version
Ubuntu 18.10 with gnome-shell xorg
## Flatpak version
Flatpak 1.0.4
## Description of the problem
All the apps I've installed with flatpak can't display all the chinese characters correctly.
Tested with: UberWriter, GNote, TODO, HomeBank. Screenshots attached.
![screenshot from 2018-10-21 22-48-21](https://user-images.githubusercontent.com/16731997/47272310-2fd9c380-d584-11e8-904d-a44762e7ecda.png)
![screenshot from 2018-10-21 22-48-44](https://user-images.githubusercontent.com/16731997/47272311-2fd9c380-d584-11e8-9461-de608a3a8e4c.png)
![screenshot from 2018-10-21 22-49-20](https://user-images.githubusercontent.com/16731997/47272312-2fd9c380-d584-11e8-8d4a-acbc32bf9a33.png)

## Steps to reproduce
Just type in "这是一个实验，看看字体有没有问题", and there will be characters missing.

**Comment by mwleeds** [comment]
This is likely a duplicate of https://github.com/flatpak/flatpak/issues/2031

**Comment by peter-jerry-ye** [comment]
@mwleeds No, I use ibus. Plus, this issue appeared after upgraded to 3.30. And I've already updated to 1.0.0 before the problem appeared.

**Comment by peter-jerry-ye** [comment]
Tested with Okular, the only app I installed with KDE runtime, and it worked well, so it seems to be a Gnome specific problem.

**Comment by peter-jerry-ye** [comment]
It seems to be a font problem. TakaoPGothic? What's that?
Logs attached.
[todo.log](https://github.com/flatpak/flatpak/files/2509408/todo.log)
[uberwriter.log](https://github.com/flatpak/flatpak/files/2509409/uberwriter.log)
